config/crowdtangle_list_example.json

```diff
@@ -0,0 +1,513 @@
+{
+    "crowdtangle_list_account_pairs": {
+        "12045542": "Pro-NLD Pages",
+        "12045541": "Pro-NLD Pages",
+        "12045540": "Pro-NLD Pages",
+        "12045538": "Pro-NLD Pages",
```

Comment:
There's a LOT of lines here that non-myanmar users will need to delete, can we make it stub data instead of real? Like `"1234": "Cats in Boxes"`. On second thought do we even need this example file at all?

Reply:
removed the file. dont think its necessary.